### PR TITLE
pallet evm weights

### PR DESCRIPTION
**Pull request by asiniscalchi**
### **PR Type**
Enhancement

___

### **Description**
- Updated benchmark dates and host information across multiple pallets.
- Adjusted weight values, proof size estimates, and execution times for various functions in parachain staking, vesting, proxy, multisig, utility, sudo, and XCMP queue pallets.
- Updated storage proof details to reflect new benchmarks.
- Modified the `WeightInfo` type for the EVM pallet configuration.

___

### **Changes walkthrough** 📝
<table><thead><tr><th></th><th align="left">Relevant files</th></tr></thead><tbody><tr><td><strong>Enhancement
</strong></td><td><table>
<tr>
  <td>
    <details>
      <summary><strong>pallet_parachain_staking.rs</strong><dd><code>Update weights and benchmarks for parachain staking pallet</code></dd></summary>
<hr>

runtime/laos/src/weights/pallet_parachain_staking.rs
<li>Updated benchmark dates and host information.<br> <li> Adjusted weight values for various functions.<br> <li> Modified proof size estimates and execution times.<br> <li> Updated storage proof details.<br>

</details>
    

  </td>
  <td><a href="https://github.com/freeverseio/laos/pull/609/files#diff-bc556c5e072eb6f13a0b4adea48f6cca41cc1233ef80b6be2f6d37fafab77f66">+204/-204</a></td>
</tr>                    

<tr>
  <td>
    <details>
      <summary><strong>pallet_vesting.rs</strong><dd><code>Update weights and benchmarks for vesting pallet</code>&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; </dd></summary>
<hr>

runtime/laos/src/weights/pallet_vesting.rs
<li>Updated benchmark dates and host information.<br> <li> Adjusted weight values for vesting functions.<br> <li> Modified proof size estimates and execution times.<br> <li> Updated storage proof details.<br>

</details>
    

  </td>
  <td><a href="https://github.com/freeverseio/laos/pull/609/files#diff-2bbbf4d5b466b46102c72df471173b5cc077fbc11cc36b6a0db9e4496501b9e5">+64/-64</a>&nbsp; </td>
</tr>                    

<tr>
  <td>
    <details>
      <summary><strong>pallet_proxy.rs</strong><dd><code>Update weights and benchmarks for proxy pallet</code>&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; </dd></summary>
<hr>

runtime/laos/src/weights/pallet_proxy.rs
<li>Updated benchmark dates and host information.<br> <li> Adjusted weight values for proxy functions.<br> <li> Modified proof size estimates and execution times.<br> <li> Updated storage proof details.<br>

</details>
    

  </td>
  <td><a href="https://github.com/freeverseio/laos/pull/609/files#diff-ae592d7ba43b53a648ed4154cbc22be7ecdab0b43718b9444733305db0b362e7">+57/-59</a>&nbsp; </td>
</tr>                    

<tr>
  <td>
    <details>
      <summary><strong>pallet_multisig.rs</strong><dd><code>Update weights and benchmarks for multisig pallet</code>&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; </dd></summary>
<hr>

runtime/laos/src/weights/pallet_multisig.rs
<li>Updated benchmark dates and host information.<br> <li> Adjusted weight values for multisig functions.<br> <li> Modified proof size estimates and execution times.<br> <li> Updated storage proof details.<br>

</details>
    

  </td>
  <td><a href="https://github.com/freeverseio/laos/pull/609/files#diff-df31f29ab6a5a60dbad552fc44cd54ff2f8b7315a0f1895b4e4e0ca3c61479c4">+39/-39</a>&nbsp; </td>
</tr>                    

<tr>
  <td>
    <details>
      <summary><strong>pallet_utility.rs</strong><dd><code>Update weights and benchmarks for utility pallet</code>&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; </dd></summary>
<hr>

runtime/laos/src/weights/pallet_utility.rs
<li>Updated benchmark dates and host information.<br> <li> Adjusted weight values for utility functions.<br> <li> Modified proof size estimates and execution times.<br>

</details>
    

  </td>
  <td><a href="https://github.com/freeverseio/laos/pull/609/files#diff-6cb936a6c4c422d45ded7c952bc026e8cec398c418cdd7690f7ea6a5cea9504f">+18/-18</a>&nbsp; </td>
</tr>                    

<tr>
  <td>
    <details>
      <summary><strong>pallet_sudo.rs</strong><dd><code>Update weights and benchmarks for sudo pallet</code>&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; </dd></summary>
<hr>

runtime/laos/src/weights/pallet_sudo.rs
<li>Updated benchmark dates and host information.<br> <li> Adjusted weight values for sudo functions.<br> <li> Modified proof size estimates and execution times.<br>

</details>
    

  </td>
  <td><a href="https://github.com/freeverseio/laos/pull/609/files#diff-eaea3924188092563f8f7e510f92a65bda98814e8e0aa61fb890f75bd675b951">+8/-8</a>&nbsp; &nbsp; &nbsp; </td>
</tr>                    

<tr>
  <td>
    <details>
      <summary><strong>cumulus_pallet_xcmp_queue.rs</strong><dd><code>Update weights and benchmarks for XCMP queue pallet</code>&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; </dd></summary>
<hr>

runtime/laos/src/weights/cumulus_pallet_xcmp_queue.rs
<li>Updated benchmark dates and host information.<br> <li> Adjusted weight values for XCMP queue functions.<br> <li> Modified proof size estimates and execution times.<br>

</details>
    

  </td>
  <td><a href="https://github.com/freeverseio/laos/pull/609/files#diff-0414a5622d2e5b3b8431f23747d7a85965b19d1ddd646ade1cfebc28e4131816">+6/-6</a>&nbsp; &nbsp; &nbsp; </td>
</tr>                    

<tr>
  <td>
    <details>
      <summary><strong>evm.rs</strong><dd><code>Update EVM pallet configuration for weight info</code>&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; </dd></summary>
<hr>

runtime/laos/src/configs/evm.rs
- Updated the `WeightInfo` type for the EVM pallet configuration.

</details>
    

  </td>
  <td><a href="https://github.com/freeverseio/laos/pull/609/files#diff-705ca1a21913654f820d021a4ab6d046d975864f252f68537584e89bcd379518">+1/-1</a>&nbsp; &nbsp; &nbsp; </td>
</tr>                    
</table></td></tr></tr></tbody></table>

___

> 💡 **PR-Agent usage**:
>Comment `/help` on the PR to get a list of all available PR-Agent tools and their descriptions

### CODE REVIEW

File: runtime/laos/src/benchmarks.rs [hunk 1]
```diff
@@ -15,19 +15,19 @@
 // along with LAOS.  If not, see <http://www.gnu.org/licenses/>.
 
 frame_benchmarking::define_benchmarks!(
-	[frame_system, SystemBench::<Runtime>]
+	// [frame_system, SystemBench::<Runtime>]
```
Comment (ccubu) [inline runtime/laos/src/benchmarks.rs:18]: On this comments, can we put this comment from the previous version explaining why we don't use this specific pallets:

``` 
 # It fails but others parachain as Moonbeam does not have this
  "frame_system" 
  # We use default weights from polkadot-sdk as benchmarking functions does not support existential deposit zero
  "pallet_balances" ```

File: runtime/laos/src/benchmarks.rs [hunk 2]
```diff
@@ -15,19 +15,19 @@
...
-	[pallet_balances, Balances]
+	// [pallet_balances, Balances]
 	[pallet_vesting, Vesting]
 	[pallet_session, SessionBench::<Runtime>] // TODO check why SessionBench::<Runtime>
 	[pallet_parachain_staking, ParachainStaking]
 	[cumulus_pallet_xcmp_queue, XcmpQueue]
 	[pallet_evm, EVM]
-	[pallet_laos_evolution, LaosEvolution]
-	[pallet_asset_metadata_extender, AssetMetadataExtender]
+	// [pallet_laos_evolution, LaosEvolution]
```
Comment (ccubu) [inline runtime/laos/src/benchmarks.rs:30]: Out of curiosity why our pallets don't have to compute weights?

Comment (asiniscalchi) [reply runtime/laos/src/benchmarks.rs:30]: it something that we are going to handle in #584

**Review by ccubu — APPROVED** [review]

**Pull request merged**